src/app/work-item/work-item-list/work-item-list-entry/work-item-list-entry.component.html

```diff
@@ -57,3 +58,51 @@
 		</div>
 	</div>
 </div>
+
+<modal #iterationAssociationModal title="Associate with Iteration">
+    <modal-content>
+        <div class="col-md-12 col-sm-12 col-xs-12">
+            <div *ngIf="isAssociated">
+                <strong>#ThisWorkItem</strong>
+                is currently associated with
+                <strong>#IterationName</strong>
+            </div>
+            <div class="form-group" [class.has-error]="validationError">
+                <div *ngIf="isAssociated" class="search-pf-input-group">
+                    <label>Reassociate <strong>#ThisWorkItem</strong> with</label>
+                    <div class="dropdown" dropdown>
+                        <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown" dropdown-open>
+                            #IterationName <span class="caret"></span>
+                        </button>
+                        <ul class="dropdown-menu" role="menu">
+                            <li role="presentation"><a role="menuitem" tabindex="-1">Itenration 0</a></li>
+                            <li role="presentation"><a role="menuitem" tabindex="-1">Itenration 1</a></li>
+                            <li role="presentation"><a role="menuitem" tabindex="-1">Itenration 2</a></li>
+                            <li role="presentation"><a role="menuitem" tabindex="-1">Itenration 3</a></li>
+                        </ul>
+                    </div>
+                </div>
+                <div *ngIf="!isAssociated" class="search-pf-input-group">
+                    <label>Associate <strong>#ThisWorkItem</strong> with</label>
+                    <div class="dropdown" dropdown>
+                        <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown" dropdown-open>
+                            Select... <span class="caret"></span>
+                        </button>
+                        <ul class="dropdown-menu" role="menu">
+                            <li role="presentation"><a role="menuitem" tabindex="-1">Itenration 0</a></li>
+                            <li role="presentation"><a role="menuitem" tabindex="-1">Itenration 1</a></li>
+                            <li role="presentation"><a role="menuitem" tabindex="-1">Itenration 2</a></li>
+                            <li role="presentation"><a role="menuitem" tabindex="-1">Itenration 3</a></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+            <div class="text-right">
```

Comment:
If we are aligning the block we should use float:right instead of text-align:right. Text align is used for aligning the  text inside the element